script.js

```diff
@@ -1,8 +1,8 @@
-const slideMenu = document.querySelector('.slide_menu');
+const slideMenu = document.querySelector('.slide-menu');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-bag/pullRequest?prid=7696171)
```suggestion
const slideMenu = document.querySelector(".slide-menu");
```